dashboard/app/models/featured_project.rb

```diff
@@ -21,10 +21,16 @@ class FeaturedProject < ApplicationRecord
   # to reflect the new table name, so an alias is used to clarify which table this ID maps to.
   alias_attribute :project_id, :storage_app_id
 
-  def featured?
+  def active?
     !featured_at.nil? && unfeatured_at.nil?
   end
 
+  def status
+    return 'active' if active?
+    return 'archived' if !featured_at.nil? && !unfeatured_at.nil?
+    'bookmarked'
```

Comment:
Do we want these statuses in an enum or constant somewhere to reference for consistency?

Reply:
Great idea - will update. Thanks!